community/kernel/src/main/java/org/neo4j/kernel/impl/factory/EditionModule.java

```diff
@@ -117,19 +130,39 @@ public static AuthManager createAuthManager( Config config, LogService logging )
         }
 
         String key = config.get( GraphDatabaseSettings.auth_manager );
+        Log authManagerLog;
+        try
+        {
+            authManagerLog = authManagerLog( config, fileSystem, jobScheduler );
+        }
+        catch ( IOException ioe ){
+            logging.getInternalLog( GraphDatabaseFacadeFactory.class ).warn( "Unable to create log for auth-manager. " +
+                    "Auth logging turned off." );
+            authManagerLog = NullLog.getInstance();
+        }
```

Comment:
I agree that it would simplify the signature of this method, but it would make calling this method more cumbersome, as you'd need to pass two parameters twice (`config` and `logging`) to the both respective create methods. The caller would then also get access to the log, which is not intended.